Nef_3/include/CGAL/Nef_3/SNC_indexed_items.h

```diff
@@ -39,7 +42,7 @@ class Index_generator {
  public:
   static int get_unique_index()
   {
-    static int unique = 0;
+    CGAL_STATIC_THREAD_LOCAL_VARIABLE(int, unique, 0);
```

Comment:
So the index will only be unique inside a thread, not globally. Can that be a problem? Should the doc warn about it? (maybe it already does, I didn't check)